spec/support/vcr_cassettes/lighthouse/benefits_claims/show/200_response.yml

```diff
@@ -242,7 +242,7 @@ http_interactions:
                            {
                               "closedDate": null,
                               "description": "This evidence request should never be surfaced to the end user.",
-                              "displayName": "Attorney Fee",
+                              "displayName": "Attorney Fees",
```

Comment:
Are these two cassettes re-recorded or did you change them manually? It can be a little risky to change something like this manually unless you're _sure_ it's accurate. i.e. tests could pass but the code could fail in real life.

Reply:
@rmtolmach as discussed in Slack, this change is harmless and the result of my inability to find a real-world response that includes this information I'm looking to filter.